packages/sitecore-jss/src/personalize/utils.test.ts

```diff
@@ -52,6 +52,13 @@ describe('utils', () => {
       const result = getPersonalizedRewriteData(pathname);
       expect(result.variantId).to.equal('');
     });
+    it('should return varinat id from any position in pathname', () => {
```

Comment:
We have unit test which covers this case:
https://github.com/Sitecore/jss/pull/1286/files#diff-1f4e67a3c64ca2fb78679219cf3a4ece8637f674e6e35e1af42290ef3daea60bR40-R44